ask-sdk-runtime/pom.xml

```diff
@@ -36,17 +36,22 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>2.10.5</version>
+            <version>2.13.2</version>
```

Comment:
Lets stick to 2.12.x version as mentioned by dependency alert

Reply:
see the latest commit